Code releases 05.20/daba1c6f-cf44-4da3-a2f2-630b862c37b5.md

```diff
@@ -119,6 +119,10 @@ $config[\Spryker\Shared\ZedNavigation\ZedNavigationConstants::ZED_NAVIGATION_CAC
 
 ## Activate Class Resolver Cache
 
+:::(Warning) (Warning)
+We do not recommend to use this feature along with PHP 7.2 due to https://bugs.php.net/bug.php?id=75765.
```

Comment:
Will this link be auto linked or do we have to use markup here?

Reply:
Done.